center/src/main/java/com/microsoft/hydralab/center/interceptor/BaseInterceptor.java

```diff
@@ -77,11 +77,11 @@ public boolean preHandle(HttpServletRequest request, HttpServletResponse respons
             }
             //invoke by browser
             if (StringUtils.isEmpty(token) || !authUtil.verifyToken(token)) {
-                if (requestURI.contains(Const.FontPath.PREFIX_PATH)) {
+                if (requestURI.contains(Const.FrontPath.PREFIX_PATH)) {
                     String queryString = request.getQueryString();
                     if (StringUtils.isNotEmpty(queryString)
-                            && queryString.startsWith(Const.FontPath.REDIRECT_PARAM)
-                            && LogUtils.isLegalStr(queryString.replace(Const.FontPath.REDIRECT_PARAM + "=", ""), Const.RegexString.URL, false)
+                            && queryString.startsWith(Const.FrontPath.REDIRECT_PARAM)
+                            && LogUtils.isLegalStr(queryString.replace(Const.FrontPath.REDIRECT_PARAM + "=", ""), Const.RegexString.URL, false)
```

Comment:
Change this FontPath in another PR, this PR is already large.

Reply:
Pruned, thx.